packages/pwa-kit-runtime/src/ssr/server/express.js

```diff
@@ -460,3 +463,102 @@ export const getRuntime = () => {
 
     return boundRuntime
 }
+
+/**
+ * Patches `res.setHeader` to ensure that the Content-Security-Policy header always includes the
```

Comment:
This comment describes _part_ of what this function does:

1. It is an express.js middleware that sets default security headers including CSP + HSTS for PWA Kit sites.
2. It patches `res.setHeader` to include defaults normally required for PWA Kit operation.